build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "delphi-webapi"
 
-version := "0.9.5-SNAPSHOT"
+version := "0.9.5.1"
```

Comment:
But in develop shouldn't this be a SNAPSHOT version.

Reply:
Ah okey, i did not get that concept. Changed it to 0.9.6-SNAPSHOT